.gitignore

```diff
@@ -110,5 +110,6 @@ release.json
 messages.mo
 
 docker/requirements-local.txt
+docker/.env-local
```

Comment:
Note: I was working with `docker-compose` and had to set my `SLACK_API_KEY` somewhere, and was afraid that I'd accidentally push it to the repo while working in `docker./env` so I added a new **gitignored** docker/.env-local